index.html

```diff
@@ -36,3 +33,158 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
         <li class="fa fa-facebook fa-2x"></li>
        </ul>
     </section>
+   </div>
+   <div class="project-section">
```

Comment:
- [ ] [OPTIONAL] Well done on implementing the changes requested by the previous reviewers. I think it's better to use the HTML semantic tag `<section>` instead of a div for the projects section. Inside you can use a div or article for the cards. To keep your webpage semantic.